app/presenters/facility_management.rb

```diff
@@ -0,0 +1,14 @@
+class FacilityManagement
```

Comment:
I just realized there's `AccountManager`, which would probably be a better home for this (with some refactoring on it).